integration_tests/suite/test_handlers.py

```diff
@@ -1570,6 +1570,9 @@ def test_vmbox_get_info(base_asset: BaseAssetLaunchingHelper):
     assert recv_vars['XIVO_MAILBOX_LANGUAGE'] == 'fr_FR'
 
 
+DATETIME_REGEX = r'(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{2,6}\+00:00)'
```

Comment:
Not sure the milliseconds are printed if they are `.00`, which may happen once in a while, making the test flaky
```suggestion
DATETIME_REGEX = r'(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\(.\d{2,6})?\+00:00)'
```